cico_build_deploy.sh

```diff
@@ -52,10 +52,20 @@ rm -rf ${TARGET_DIR}/
 docker build -t ${BUILDER_IMAGE} -f Dockerfile.build .
 
 mkdir ${TARGET_DIR}/
+mkdir ${TARGET_DIR}/images
+#This sets the permissions on target dir since this script may be invoked as root, but container is not.
+chmod -R 0777 ${TARGET_DIR}/
```

Comment:
I know it may look like nitpicking, but it seems to me you forgot to delete this line as the same invocation is now (correctly) on L66. :)